src/ListArray.java

```diff
@@ -0,0 +1,158 @@
+public class ListArray {
+
+    private int length;
+    private int[] content;
+
+    public ListArray(int length, int[] contents){
+        this.length = length;
+        this.content = new int[length];
+        for(int i = 0; i<length-1; i++){
+            this.content[i] = contents[i];
+        }
+//        System.out.println("Array generated with both parameters: L: " + this.length + "  C:" + this.content[0]+"\n");
+    }
+
+    public ListArray(int length){
+        this.length = length;
+        this.content = new int[length];
+//        System.out.println("Array generated with one length parameter: L: " + this.length + "  C:" + this.content[0]+"\n");
+    }
+
+    public ListArray(){
+        this.length = -1;
+//        System.out.println("Contentless ListArray generated... Why?"+"\n");
+    }
+
+    public ListArray(int[] contents){
+
```

Comment:
You don't need to worry about this.  The `add()` function will allow you to add items to the `ArrayList`